pkg/accelerator/accelerator_linux.go

```diff
@@ -60,13 +62,69 @@ func (i *Info) load() error {
 		if !isAccelerator(device) {
 			continue
 		}
-		accelDev := &AcceleratorDevice{
-			Address:   device.Address,
-			PCIDevice: device,
+		if len(i.DiscoveryFilters) > 0 {
```

Comment:
the way I envisioned is that the user should be able to feed more filters in addition to the unavoidable buliltin
```isAccelerator := func(dev *pci.Device) bool {}```

probably using a different interface to maximize the safety: user-provided filters should never be allowed to modify the `pci.Devices` passed to them (or, equivalently, any change made should be discarded)